regulation-worker/internal/client/schema.go

```diff
@@ -6,7 +6,7 @@ type jobSchema struct {
 	UserAttributes []userAttributesSchema `json:"userAttributes"`
 }
 
-type statusJobSchema struct {
+type StatusJobSchema struct {
```

Comment:
see if we do not need to make this Exportable if this is just being used in client package

Reply:
will look for alternative